app/controllers/concerns/hyrax/works_controller_behavior.rb

```diff
@@ -142,6 +142,13 @@ def iiif_manifest_builder
         self.class.iiif_manifest_builder
       end
 
+      def iiif_manifest_presenter
+        IiifManifestPresenter.new(curation_concern_from_search_results).tap do |p|
```

Comment:
Putting a pin in the `tap` call. I'm curious if we can instead expect a block for initialization (or even pass those parameters on initialization). Regardless, this is not a blocker. And I'll find out more in later files.

Reply:
a Builder might be a thing, but i'm not keen on messing with the initializer, since it's provided by `Draper`.